ui/package.json

```diff
@@ -36,7 +36,7 @@
     "craco-antd": "1.11.0",
     "formik": "^2.0.0",
     "history": "^4.7.2",
-    "immutable": "^3.8.2",
+    "immutable": "4.0.0-rc.10",
```

Comment:
rc?

Reply:
yeah, it's fine it has been `rc` for so long that I think it's stable :)